src/Basque/Mews.Fiscalizations.Basque/Model/ErrorCode.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Mews.Fiscalizations.Basque.Model
+{
+    // Reference: https://www.gipuzkoa.eus/documents/2456431/28540714/cas+01+TICKETBAI+ALTA+2.1.pdf/710e8dd2-825e-5fd0-d825-a7be669e624e.
+    public enum ErrorCode
+    {
+        // Sender certificate must be approved, not revoked and not expired (more than one month).
+        RevokedOrExpiredCertificate = 001,
+
+        // The file must comply with the XSD schema.
```

Comment:
I wouldn't have these basic comments that don't actually help with anything. But I get that you want to have a comment for everything. I don't mind.

Reply:
Well, im just translating whats in the docs and adding a comment, its won't hurt :D might come handy one day.